packages/frontend/src/features/catalog/components/CatalogFieldListItem.tsx

```diff
@@ -22,40 +22,58 @@ export const CatalogFieldListItem: FC<React.PropsWithChildren<Props>> = ({
 
     return (
         <>
-            <Group
-                noWrap
-                py="xs"
+            <UnstyledButton
                 sx={(theme) => ({
                     borderRadius: theme.radius.sm,
-                    padding: theme.spacing.md,
                     backgroundColor: hovered
                         ? theme.colors.gray[1]
-                        : theme.colors.gray[0],
-                    border: isSelected
-                        ? `2px solid ${theme.colors.blue[6]}`
-                        : undefined,
+                        : 'transparent',
+                    border: `2px solid ${
+                        isSelected ? theme.colors.blue[6] : 'transparent'
+                    }`,
                 })}
                 onMouseEnter={() => setHovered(true)}
                 onMouseLeave={() => setHovered(false)}
                 onClick={onClick}
+                py="xs"
             >
-                <Group noWrap spacing="xs">
-                    <MantineIcon
-                        icon={IconAbc}
-                        color="gray"
-                        size="lg"
-                    ></MantineIcon>
-                </Group>
                 <Box miw={150}>
-                    <Highlight
-                        highlight={searchString}
-                        highlightColor="violet"
-                        fw={600}
+                    <Group
+                        spacing="xs"
+                        noWrap
+                        w="fit-content"
+                        px="xs"
+                        sx={(theme) => ({
+                            border: `1px solid ${theme.colors.gray[2]}`,
+                            borderRadius: theme.radius.md,
+                        })}
                     >
-                        {field.name || ''}
-                    </Highlight>
+                        <MantineIcon
+                            icon={
+                                // TODO: Add icon for field type and for subtype
+                                field.fieldType === FieldType.DIMENSION
+                                    ? IconAbc
+                                    : IconNumber
```

Comment:
Instead of number, should we use the same one we use in explores, filters, etc? `Icon123`

Reply:
Absolutely, but these icons will be replaced tomorrow, so it's very very temporary 👍